ddtrace/pin.py

```diff
@@ -56,6 +56,25 @@ def __repr__(self):
         return "Pin(service=%s, app=%s, app_type=%s, tags=%s, tracer=%s)" % (
             self.service, self.app, self.app_type, self.tags, self.tracer)
 
+    @staticmethod
+    def find(*objs):
```

Comment:
@Kyle-Verhoog and @palazzem 

I found myself writing a helper `get_inherited_pin()` with the idea that there was an order to which I would look-up a `Pin`, either in the `wrapped`, `instance`, or maybe `flask.current_app`, etc.

Not sure if this is a public API we care to expose, seemed "harmless" to add, and potentially useful as we broaden the discussion about pin/config overriding.

Reply:
totally happy to either change my code to not need something like this, or to move it back into a flask specific helper.

Reply:
I think the API is fine to have on the Pin. However, I think we can keep it private for now since we do document the Pin publicly: http://pypi.datadoghq.com/trace/docs/advanced_usage.html#pin. 

But we are going to be deprecating the public Pin api soon anyway.